streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -56,21 +56,21 @@
      *          |            |              |     |
      *          |            v              |     |
      *          |     +------+--------+     |     |
-     *          |     | Suspended (3) | <---+     |    //TODO Suspended(3) could be removed after we've stable on KIP-429
-     *          |     +------+--------+           |
-     *          |            |                    |
-     *          |            v                    |
-     *          |      +-----+-------+            |
-     *          +----> | Closed (4)  | -----------+
+     *          +---->| Suspended (3) | ----+     |    //TODO Suspended(3) could be removed after we've stable on KIP-429
```

Comment:
Why remove the `<` arrow? We can still transit from `RESTORING` to `SUSPENDED`.

Super-nit: `+---->| Suspended` -> `+---> | Suspended`

Reply:
The diff makes it hard to tell, but I "merged" the path to SUSPENDED from CREATED and RESTORING. I find it a bit easier to follow when all the arrows are unidirectional